pkg/workload/workload.go

```diff
@@ -572,6 +586,15 @@ func IsEvictedByPodsReadyTimeout(w *kueue.Workload) (*metav1.Condition, bool) {
 	return cond, true
 }
 
+// IsUnsetRequeuedByPodsReadyTimeout returns true if the workload is unset requeued by pods ready timeout.
+func IsUnsetRequeuedByPodsReadyTimeout(w *kueue.Workload) (*metav1.Condition, bool) {
+	cond := apimeta.FindStatusCondition(w.Status.Conditions, kueue.WorkloadRequeued)
+	if cond == nil || cond.Status != metav1.ConditionFalse || cond.Reason != kueue.WorkloadEvictedByPodsReadyTimeout {
+		return nil, false
+	}
+	return cond, true
```

Comment:
```suggestion
	return cond != nil && cond.Status == metav1.ConditionFalse && cond.Reason == kueue.WorkloadEvictedByPodsReadyTimeout
```
as above, much easier to read

Reply:
Done